developer-docs/account/README.md

```diff
@@ -206,23 +206,23 @@ Note: using [docker compose file](../../docker-compose.yaml) to start the servic
 The following command will start all of the necessary containers for the account service to run in development mode.
 
 ```bash
-make restart-chain-docker
+./scripts/account/restart-chain-docker
```

Comment:
Opted to specify the scripts that are specific to account service, rather than propagate a make command, which would need to be duplicated for each service app.